pkg/app/config.go

```diff
@@ -8,12 +8,12 @@ import (
 
 	"github.com/openshift/ansible-service-broker/pkg/apb"
 	"github.com/openshift/ansible-service-broker/pkg/broker"
-	"github.com/openshift/ansible-service-broker/pkg/dao"
+	"github.com/openshift/ansible-service-broker/pkg/clients"
 )
 
 type Config struct {
 	Registry   apb.RegistryConfig
-	Dao        dao.Config
+	Dao        clients.EtcdConfig
```

Comment:
This will mean that if we ever want to add some configuration to the dao that is not etcd related we'd have to redo this. dao.Config was easier so we could configure all of the dao. I see how you arrived here since the only configuration items on the dao were etcd related, but it paints in a corner of only being etcd.